appveyor.yml

```diff
@@ -1,21 +1,23 @@
 version: 4.1.0.{build}
+image: Visual Studio 2019
```

Comment:
looks like changing the base image from 2015 to 2019 made the tests excruciatingly slow... and we also have what appear to be valid failures:

```
Failures:
  1) dependency loader FileBased module loader when parsing files from disk when loading files from disk should always read files as UTF-8
     Failure/Error: raise 'This test must be run in a codepage other than 65001 to validate behavior'
     RuntimeError:
       This test must be run in a codepage other than 65001 to validate behavior
     # ./spec/unit/pops/loaders/dependency_loader_spec.rb:88:in `block (5 levels) in <top (required)>'
     # ./spec/spec_helper.rb:180:in `block (2 levels) in <top (required)>'
     # util/rspec_runner:44:in `run'
     # util/rspec_runner:59:in `<main>'
  2) dependency loader FileBased module loader when parsing files from disk when loading files from disk currently ignores the UTF-8 BOM (Byte Order Mark) when loading module files
     Failure/Error: raise 'This test must be run in a codepage other than 65001 to validate behavior'
     RuntimeError:
       This test must be run in a codepage other than 65001 to validate behavior
     # ./spec/unit/pops/loaders/dependency_loader_spec.rb:106:in `block (5 levels) in <top (required)>'
     # ./spec/spec_helper.rb:180:in `block (2 levels) in <top (required)>'
     # util/rspec_runner:44:in `run'
     # util/rspec_runner:59:in `<main>'
Finished in 1 minute 35.53 seconds (files took 8.02 seconds to load)
  1) Puppet::Util#withenv on Windows works around Ruby bug 8822 (which fails to preserve UTF-8 properly when accessing ENV) (Ruby >= 2.3.x) 
     Failure/Error: raise 'This test requires a non-UTF8 codepage' if Encoding.default_external == Encoding::UTF_8
     RuntimeError:
       This test requires a non-UTF8 codepage
     # ./spec/unit/util_spec.rb:153:in `block (3 levels) in <top (required)>'
     # ./spec/spec_helper.rb:180:in `block (2 levels) in <top (required)>'
     # util/rspec_runner:44:in `run'
     # util/rspec_runner:59:in `<main>'
Finished in 1 minute 0.89 seconds (files took 4.9 seconds to load)
  1) Puppet::Pops::Parser::Lexer2 when lexing files from disk should always read files as UTF-8
     Failure/Error: raise 'This test must be run in a codepage other than 65001 to validate behavior'
     RuntimeError:
       This test must be run in a codepage other than 65001 to validate behavior
     # ./spec/unit/pops/parser/lexer2_spec.rb:926:in `block (3 levels) in <top (required)>'
     # ./spec/spec_helper.rb:180:in `block (2 levels) in <top (required)>'
     # util/rspec_runner:44:in `run'
     # util/rspec_runner:59:in `<main>'
Finished in 1 minute 13.8 seconds (files took 4.77 seconds to load)
```